en/docs/deploy/depoy-your-component.md

```diff
@@ -1,3 +1,21 @@
 # Deploy Your Component
 
-Content of https://wso2.com/choreo/resources/how-to-deploy-your-api-using-choreo/ will be extracted in the form of a how-to guide.
\ No newline at end of file
+Depoying your component on Choreo makes your component invocable. Once you have designed, tested, and committed the REST API, you can proceed to to deploy it.
+
+To deploy a component, follow the steps below:
+
+1. Click the **Deploy** icon. Then in the **Build Area** card, click **Deploy**.
+
+    ![Deploy componnet](../../assets/img/tutorials/rest-api/deploy-api.png){.cInlineImage-full}
```

Comment:
Let's change "componnet" to "component".

Reply:
done

Reply:
This still needs to be fixed. Now it is "componet"

Reply:
done